src/components/explorer/ExplorerDatasetSearch.js

```diff
@@ -59,27 +62,20 @@ const ExplorerDatasetSearch = () => {
     };
 
     const resetPageNumber = () => {
```

Comment:
Function name no longer describes its purpose, could be renamed to `performSearch`

Reply:
Yes, better name. Changed!